src/components/step/styles.ts

```diff
@@ -0,0 +1,47 @@
+import styled from "styled-components";
+import Button from "../button/button";
+import { colors } from "src/constants";
+
+export const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
```

Comment:
We have border box like default value, remove it from styles, use it only when it need. . .